src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const object = {};
+  const arrayOfProperties = sourceString.split(';')
+    .map(el => el.trim()).filter(element => element !== '');
+  const property = arrayOfProperties.map(el => el.split(':'));
+  const trimmedValues = property.map(el => el[1].trim());
+  const trimmedKeys = property.map(el => el[0].trim());
```

Comment:
Роби відступи між змінними, щоб їх було простіше читати
```suggestion
  const arrayOfProperties = sourceString.split(';')
    .map(el => el.trim()).filter(element => element !== '');
    
  const property = arrayOfProperties.map(el => el.split(':'));
  
  const trimmedValues = property.map(el => el[1].trim());
  
  const trimmedKeys = property.map(el => el[0].trim());
```